source/js/carousel.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable no-use-before-define */
 
 // get the carousel images
-const images = Array.from(document.querySelectorAll('.carousel-item img'));
+const images = Array.from(document.querySelectorAll('.carousel-item img:not([style])'));
```

Comment:
some magic here - how are we picking the images?

Reply:
No magic just standard selector technique acquired from css

Reply:
but why the ":not([style])" - please explain in code.